quickwit/quickwit-indexing/src/source/kafka_source.rs

```diff
@@ -853,7 +853,7 @@ mod kafka_broker_tests {
                     Duration::from_secs(1),
                 )
                 .await
-                .map(|(partition, offset)| (id, partition, offset))
+                .map(|delivery| (id, delivery.partition, delivery.offset)) 
```

Comment:
Now it's fmt that isn't happy because of the trailing space. You can run `cargo fix` as advised in the [contribution guide](https://github.com/quickwit-oss/quickwit/blob/main/CONTRIBUTING.md) to prevent these issues. Thanks!